lib/filter/iir.c

```diff
@@ -1,7 +1,7 @@
 #include <math.h>
 #include <string.h>
 
-#include "iir.h"
+#include "filter/iir.h"
```

Comment:
I'd love to see more <> on those public header includes

Reply:
Fixed in (b9f44270443ebf21de4eb6dd9f8a3c6f864f7794 might be rebased at any point). I also added some automation to enforce that those headers stay coherent.